code/controllers/subsystem/SStimer.dm

```diff
@@ -502,14 +502,15 @@ SUBSYSTEM_DEF(timer)
   * If the timed event is tracking client time, it will be added to a special bucket.
   */
 /datum/timedevent/proc/bucketJoin()
-	// Generate debug-friendly name for timer
+	// Generate debug-friendly name for timer, uncomment for explicit debug
+	/*
 	var/static/list/bitfield_flags = list("TIMER_UNIQUE", "TIMER_OVERRIDE", "TIMER_CLIENT_TIME", "TIMER_STOPPABLE", "TIMER_NO_HASH_WAIT", "TIMER_LOOP")
 	name = "Timer: [id] (\ref[src]), TTR: [timeToRun], wait:[wait] Flags: [jointext(bitfield2list(flags, bitfield_flags), ", ")], \
```

Comment:
Since by default, it will have no name, could you please set a default value to the name variable for timers? Something like "Unnamed timer"

Also,
calling getTimerInfo() on /datum/timedevent/can_vv_get() and returning ..() would be good imo, so that if an admin needs to view a timer, they can actually see the name for it.

Reply:
Seems like there is a small issue with that (not giving name on first visit) but refreshing VV gives right results
![изображение](https://github.com/ParadiseSS13/Paradise/assets/73733747/4e09de6a-f25e-4318-a3c8-d28ad13afbf3)
![изображение](https://github.com/ParadiseSS13/Paradise/assets/73733747/477baa88-3513-4713-9022-9929d77206c3)

Reply:
This is good, can_vv_get comes after when its assigned apparently, which I'm fine with